LinkitOne/bluetooth.ino

```diff
@@ -0,0 +1,63 @@
+""" This program scans for Bluetooth devices in the vicinity of LinkIt One Microcontroller. It scans once in every three seconds for such devices""""
+#include <LBT.h>
+#include <LBTClient.h>
+#include <SPI.h>
+#include <LSD.h>
+#include <Time.h>
+""" Declaring a file to be stored on the sd card """
+LFile mbt;
+
+void setup() {
+  // put your setup code here, to run once:
+    Serial.begin(9600);
+    Serial.print("Initializing SD card...");
+    LSD.begin();
+
+    mbt = LSD.open("mac.txt", FILE_WRITE);
+    if (!mbt)
+
```

Comment:
Follow same indenting style everywhere, one place has 
loop() {
while other has
if() 
{

Best way to do it is, always auto-indent using Tools->Auto Format